File: Clientes/wwwroot/js/Cliente.js
var datatable;

$(document).ready(function () {
    loadDataTable();



    function LoadDatable() {
        datatable = $('#tblData').DataTable({
            "ajax": {
                "url:": "/Cliente/ObtenerTodos"
            },
            "Columns": [

                { "data": "nombre", "width": "10 %" },
                { "data": "cedula", "width": "10 %" },
                { "data": "telefono", "width": "10 %" },
                { "data": "direccion", "width": "10 %" },
                {
                    "data": "estado", "render": function (data) {
                        if (data == true) {
                            return "Activo"
                        }
                        else {
                            return "Inactivo";
                        }

                    }, "width": "20%",

                }

            ]
        })
    }
})

File: Clientes/Models/Cliente.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace Clientes.Models
{
    public class Cliente
    {
        [Key]
        public int Id { get; set; }
        [Required]
        [DisplayName("Nombre Cliente")]
        [MinLength(1), MaxLength(40)]

        public string Nombre { get; set; }
        [Required]
        [DisplayName("Cedula")]
        [MinLength(1), MaxLength(30)]
        public string Cedula { get; set; }
        [Required]
        [DisplayName("Telefono")]
        [MinLength(1) ,MaxLength(20)]
        public string Telefono { get; set; }
        [Required]
        [DisplayName("Direccion")]
        [MinLength(1), MaxLength(80)]

        public string Direccion { get; set; }
        [Required]
        [DisplayName("Estado")]
        [MinLength(1), MaxLength(20)]

        public status Estado { get; set; }



    }
    public enum status
    {
        Inactivo,
        Activo
    }

}
